sdk/keyvault/Azure.Security.KeyVault.Administration/src/BackupOperation.cs

```diff
@@ -36,20 +36,20 @@ public BackupOperation(string id, KeyVaultBackupClient client)
             Argument.AssertNotNull(client, nameof(client));
 
             _client = client;
-            _value = new FullBackupDetailsInternal(string.Empty, string.Empty, null, null, null, id, string.Empty);
+            _value = new FullBackupDetailsInternal(null, null, null, null, null, id, string.Empty);
         }
 
         /// <summary>
         /// Initializes a new instance of a BackupOperation.
         /// </summary>
         /// <param name="client">An instance of <see cref="KeyVaultBackupClient" />.</param>
         /// <param name="response">The <see cref="ResponseWithHeaders{T, THeaders}" /> returned from <see cref="KeyVaultBackupClient.StartBackup(Uri, string, CancellationToken)"/> or <see cref="KeyVaultBackupClient.StartBackupAsync(Uri, string, CancellationToken)"/>.</param>
-        internal BackupOperation(KeyVaultBackupClient client, ResponseWithHeaders<FullBackupDetailsInternal, ServiceFullBackupHeaders> response)
+        internal BackupOperation(KeyVaultBackupClient client, ResponseWithHeaders<ServiceFullBackupHeaders> response)
         {
             _client = client;
             _response = response;
             _retryAfterSeconds = response.Headers.RetryAfter;
-            _value = response.Value ?? throw new InvalidOperationException("The response does not contain a value.");
+            _value = new FullBackupDetailsInternal(null, null, null, null, null, response.Headers.JobId(), string.Empty);
```

Comment:
I still don't understand why this is a sentinel. `Value` should throw unless the operation has completed successfully (based on guidelines) - sort of like how `Nullable<T>.Value` throws if `HasValue` is false. For one, I see no indication this sentinel would cause `Value` to throw. Even if you did, this should be a singleton because it should be immutable since you'd be deserializing the details when you actually got them back from the service.

Reply:
Sorry, that was a miss on my part.

Reply:
FWIW, I should clarify my concern is less about the sentinel itself (though, the extra - but rare - allocation is unnecessary) but more that `Value` shouldn't return anything unless the operation completed successfully; otherwise, it should throw. See the delete and restore pseudo-LROs in the other three KV libraries for an example.

Reply:
`Value` throw semantics are implemented in the property getter.

Reply:
Per the offline discussion, the null value properties need to be properly handled in the `Value` getter.